packages/react-interactions/events/src/dom/PressLegacy.js

```diff
@@ -570,7 +570,7 @@ const pressResponderImpl = {
                 metaKey,
                 shiftKey,
               } = (nativeEvent: MouseEvent);
-              if (nativeEvent.key === ' ') {
+              if (isValidKeyboardEvent(nativeEvent)) {
```

Comment:
This condition is already met on L566. This changes means the means that the `else if` clause on L575 will never be reached

Reply:
Yeah, let me revise this somewhat.